lib/dataElementListener.js

```diff
@@ -275,6 +279,9 @@ class DataElementImporter extends SHRDataElementParserListener {
         value.addConstraint(new ValueSetConstraint(vs, path, strength));
       } else if (cst.elementCodeValueConstraint()) {
         const code = this.processCodeOrFQCode(cst.elementCodeValueConstraint().codeOrFQCode());
+        if (idText == 'Value') {
```

Comment:
I think this may be too narrow.  For example, what if there is a _field_ that has a value of `concept`?
```
DataElement: ElementWithCodedElementField
Property: CodedElement
CodedElement = FOO#bar
```
In this case, we still want the path (on `CodedElement`'s `CodeConstraint`) to be `[concept]` -- even though we're not using the `Value` keyword.

In addition, you're not looking at the `path` at all.  If there is already a `path` on it, then `value` is actually irrelevant.  We're interested in what's at the tail end of the path.

I think the general rule is that if there is no path and the value identifier is not `concept`, _or_ there is a path, but the end of it is not `concept`, then you should add `concept` to the path. If, on the other hand, `concept` is already there, you don't want to add it.

This would make the check something more like:
```js
const conceptIdenfier = new PrimitiveIdentifier('concept');
const tailIdentifier = path.length > 0 ? path[path.length-1] : value.identifier;
if (!conceptIdentifier.equals(tailIdentifier)) {
  path.push(conceptIdentifier);
}